src/components/product_menu_components.rs

```diff
@@ -0,0 +1,89 @@
+use maud::{html, Markup};
+
+use crate::dso::product::Product;
+
+pub fn products_table_component(products: &[Product]) -> Markup {
```

Comment:
```suggestion
pub fn products_table(products: &[Product]) -> Markup {
```

Reply:
But it is a component. If we should not have component in the name `clickable_products_table_component` should be renamed as well.